sections/api/typescript.md

```diff
@@ -85,14 +87,13 @@ React-Native:
 ```jsx
 // styled-components.ts
 import * as styledComponents from "styled-components/native";
-
-import ThemeInterface from "./theme";
+import DefaultTheme from "styled-components/native";
 
 const {
   default: styled,
   css,
   ThemeProvider
-} = styledComponents as styledComponents.ReactNativeThemedStyledComponentsModule<ThemeInterface>;
+} = styledComponents as styledComponents.ReactNativeThemedStyledComponentsModule<DefaultTheme>;
```

Comment:
I'm not clear on why we need this code...

the `*.d.ts` file augmenting `styled-components/native` should be enough

Reply:
Tested it and this whole code snippet seems indeed useless, the declaration file is all we need.
I removed it from the page altogether.